packages/create-cloudflare/src/helpers/command.ts

```diff
@@ -18,12 +18,14 @@ type Command = string | string[];
 
 type RunOptions = {
 	startText?: string;
-	doneText?: string;
+	doneText?: string | ((result: string) => string);
 	silent?: boolean;
 	captureOutput?: boolean;
 	useSpinner?: boolean;
 	env?: NodeJS.ProcessEnv;
 	cwd?: string;
+	transform?: (output: string) => string;
+	fallback?: (error: unknown) => string;
```

Comment:
I think it would be more descriptive to name these `transformOutput` and `fallbackOutput`

Reply:
OK